cmd/build/v2/services.go

```diff
@@ -52,7 +54,15 @@ func (bc *OktetoBuilder) GetServicesToBuild(ctx context.Context, manifest *model
 	}
 	close(toBuild)
 
+	// create a spinner to be loaded while checking if images needs to be built
+	s := spinner.New(spinner.CharSets[0], 100*time.Millisecond)
+	s.Color("blue")
+
 	if len(toBuild) == 0 {
+		s.Prefix = "Building images..."
+		s.Start()
+		time.Sleep(5 * time.Second)
```

Comment:
This stops the application for `5 Seconds`. Could you move the creation of the spinner to line 40 just before `// check if images are at registry (global or dev) and set envs or send to build` comment?

Reply:
The spinner is taking time to load. How do we make sure the spinner runs as soon as the `if` block is hit?

Reply:
I see so this was the catch! I'd be more cautious from next time.